libraries/MultiThreading/examples/BasicMultiThreading/BasicMultiThreading.ino

```diff
@@ -0,0 +1,111 @@
+/* Basic Multi Threading Arduino Example
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+*/
+// Please read file README.md in the folder containing this example.
+
```

Comment:
This example is not working for me on ESP32-S3 devkitC. Can you please test that? 
Analog pin was changed of course.

```
Guru Meditation Error: Core  1 panic'ed (Unhandled debug exception). 
Debug exception reason: Stack canary watchpoint triggered (Analog Read) 

Backtrace: 0x4037d1bc:0x3fced4e0 0x4037bdfd:0x3fced520 0x4037a71c:0x3fced550 0x4037a712:0xa5a5a5a5 |<-CORRUPTED
```